### PR TITLE
Script to setup dev cassandra with sample data

### PR DESCRIPTION
**Pull request by matosf**
Created this script because the manual process was taking too long if repeated.
Maybe you find it useful.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -48,48 +48,7 @@ Run a development Cassandra node:
...
 
-    $ docker run --name dev-cassandra -d -p 9042:9042 cassandra:2.1
-
-Insert schema (you might need to wait a few seconds for Cassandra to boot):
-
-.. code-block:: bash
-
-    $ docker run -i --link dev-cassandra:cassandra --rm cassandra:2.1 cqlsh cassandra < schema.cql
-
-General cqlsh access to your dev instance:
```
Comment (hjacobs) [inline README.rst]: I think we should leave this "general cqlsh access" part in the README (after ./setup-dev-cassandra.sh) so people unfamiliar with Docker/Cassandra now how to look into it without digging into the shell script :smile:

File: setup-dev-cassandra.sh [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+# username/password: test0/test0
+echo "INSERT INTO provider.user \
+    (username, realm, password_hashes, scopes) VALUES \
+    ('test0', '/services', { {password_hash: '"'$2b$04$0PzwhGVD9MYyXd9sqtf/dOSgN1PC18dSWEliTQdUMT3hJztlvW3Em'"', created: 1457044516, created_by: 'test'} }, {'uid': 'true'});" >> sample-data.cql
+
+docker run -i --link ${CONTAINER_NAME}:cassandra --rm cassandra:2.1 cqlsh cassandra < sample-data.cql
+
+rm -f sample-data.cql
+
+echo "Done!"
```
Comment (hjacobs) [inline setup-dev-cassandra.sh]: Or maybe better: output the cqlsh to get access here, something like:

```
To access the local development Cassandra cluster, use:
...
```

**Comment by matosf** [comment]
Pushed requested changes.

**Comment by hjacobs** [comment]
@matosf sorry, my comment was probably misleading: I just meant keeping the "how to log in to cqlsh" part, not having "manual" and "automatic" steps duplicated.

I would completely remove the "manual" steps (as you did in first iteration), I would just show the user how to log in to cqlsh afterwards (e.g. to do extra setup steps or just look around).

**Comment by matosf** [comment]
No problem. Is it OK now?

**Comment by hjacobs** [comment]
:+1:

**Pull request merged**